app/static/js/grid_scripts.js

```diff
@@ -0,0 +1,23 @@
+document.addEventListener(
+    'DOMContentLoaded',
+    function () {
+        var all_days = document.querySelectorAll('.day'), i;
+        for (i = 0; i < all_days.length; ++i) {
+            all_days[i].onclick = function () {
+                var daily_event = document.querySelector("#day-display");
+                if (daily_event.style.flex === '0 1 30%') {
```

Comment:
Use classes, and change the code here to `element.classList.toggle('className')`.

Add the style to the CSS. Avoid mixing code and styling.